docs/package/gluon-scheduled-domain-switch.rst

```diff
@@ -21,18 +21,26 @@ domain_switch : optional (needed for domains to switch)
         - amount of time without reachable gateway to switch unconditionally
     switch_time :
         - UNIX epoch after which domain will be switched
-    connection_check_targets :
-        - array of IPv6 addresses which are probed to determine if the node is
-	  connected to the mesh
+
+check_connection :
+    targets :
+        - array of IPv6 addresses which are probed to determine if the node can
+        reach any time-servers to acquire the correct time
```

Comment:
please add that the check happens using the ICMP protocol via ping.

Reply:
please reword such that it gets clearer that a node is considered online whenever at least one of those hosts can be reached.